website/integrations/index.mdx

```diff
@@ -1,15 +1,28 @@
 ---
 title: Integrations overview
+sidebar_label: Overview
 ---
 
-There are two main types of integrations with authentik: **Applications** and **Sources**.
+## What is an integration?
 
-## Applications
+An integration is a how authentik connects to third-party applications, directories, and other identity providers.
+Integrations are categorized into two categories: **Applications** and **Sources**.
 
-authentik integrates with many applications. For a full list, and to learn more about adding documentation for a new application, refer to the [Applications](../integrations/services/index.mdx) documentation
 
-## Sources
+### Applications
 
-In addition to applications, authentik also integrates with external sources, including federated directories like Active Directory and through protocols such as LDAP, OAuth, SAML, and SCIM sources. Sources are a way for authentik to use external credentials for authentication and verification. Sources in authentik can also be used for social logins, using external providers such as Facebook, Twitter, etc.
+Applications include vendor tools such as Google Workspace, GitHub, Slack, or AWS. These applications can be integrated with authentik to provide single sign-on capabilities to
 
-To learn more, refer to the [Sources](https://docs.goauthentik.io/docs/users-sources/sources/index) documentation.
+If you're looking to integrate an application that isn't listed, authentik can be configured to work with most applications that support authentication protocols such as [SAML](../docs/add-secure-apps/providers/saml/index.md), [OAuth and OpenID Connect](../docs/add-secure-apps/providers/oauth2/index.mdx).
+
+To learn more, refer to the [Applications](../integrations/services/index.mdx) page.
+
+
```

Comment:
```suggestion
```

Reply:
Newline is being added by the formatter.